proton_driver/columns/util.py

```diff
@@ -0,0 +1,61 @@
+
+def get_inner_spec(column_name, spec):
+    brackets = 0
+    offset = len(column_name)
+
+    for i, ch in enumerate(spec[offset:], offset):
+        if ch == '(':
+            brackets += 1
+
+        elif ch == ')':
+            brackets -= 1
+
+        if brackets == 0:
+            break
+
+    return spec[offset + 1:i]
+
+
+def get_inner_columns(spec):
+    brackets = 0
+    column_begin = 0
+
+    columns = []
```

Comment:
I think the name is a little confusing. In typical usage, caller will pass full spec. The func will get inner content and then parse the columns.

...
inner = get_inner_spec(spec)
for i, x in enum(inner + ','):
...